.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Mark or close stale issues and PRs"
+on:
+  schedule:
+    # Run the stalebot every day at 8pm UTC
+    - cron: "00 20 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write # for actions/stale to close stale issues
+      pull-requests: write # for actions/stale to close stale PRs
+    runs-on: ubuntu-20.04
+    if: github.repository == 'alluxio/alluxio'
+    steps:
+      - uses: actions/stale@v6
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 60
+          days-before-pr-stale: 30
+          days-before-close: 7
+          days-before-pr-close: 14
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had recent activity.
+            It will be closed in a week if no further activity occurs.
+            Thank you for your contributions.
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had
+            recent activity. It will be closed in two weeks if no further activity occurs.
+            Thank you for your contributions.
+          close-issue-message: >
+            This issue has been automatically closed due to inactivity.
+            Please re-open if this still requires investigation.
+          close-pr-message: >
+            This pull request has been automatically closed due to inactivity.
+            Please re-open if these changes are still required.
+          stale-pr-label: "stale"
+          stale-issue-label: "stale"
+          exempt-issue-labels: "keepalive,priority-high"
```

Comment:
not `priority-medium`?

Reply:
no, priority-medium can be closed. Only priority high. And two month + 1 week is long enough for a issue

Reply:
issue can be reopen manually if needed